py4DSTEM/process/polar/polar_analysis.py

```diff
@@ -596,18 +600,298 @@ def calculate_FEM_local(
     --------
     self: PolarDatacube
         Polar datacube used for measuring FEM properties.
+    use_median: Bool
+        Use median instead of mean for statistics.
 
     Returns
     --------
-    radial_avg: np.array
-        Average radial intensity
-    radial_var: np.array
-        Variance in the radial dimension
+    local_radial_mean: np.array
+        Average radial intensity of each probe position
+    local_radial_var: np.array
+        Variance in the radial dimension of each probe position
+
+    """
 
+    # init radial data arrays
+    self.local_radial_mean = np.zeros(
+        (
+            self._datacube.shape[0],
+            self._datacube.shape[1],
+            self.polar_shape[1],
+        )
+    )
+    self.local_radial_var = np.zeros(
+        (
+            self._datacube.shape[0],
+            self._datacube.shape[1],
+            self.polar_shape[1],
+        )
+    )
 
+    # Compute the radial mean and standard deviation for each probe position
+    for rx, ry in tqdmnd(
+        self._datacube.shape[0],
+        self._datacube.shape[1],
+        desc="Radial statistics",
+        unit=" probe positions",
+        disable=not progress_bar,
+    ):
+        im = self.data[rx, ry]
+
+        if use_median:
+            im_mean = np.ma.mean(im, axis=0)
+            im_var = np.ma.mean((im - im_mean) ** 2, axis=0)
+        else:
+            im_mean = np.ma.median(im, axis=0)
+            im_var = np.ma.median((im - im_mean) ** 2, axis=0)
```

Comment:
Are these flipped?
```suggestion
        if use_median:
            im_mean = np.ma.median(im, axis=0)
            im_var = np.ma.median((im - im_mean) ** 2, axis=0)
        else:
            im_mean = np.ma.mean(im, axis=0)
            im_var = np.ma.mean((im - im_mean) ** 2, axis=0)
```

Reply:
Good catch!   Yes it was flipped - I think I changed from use_mean to use_median since I think the default assumption is mean.